Snowflake.Data/Core/RestResponse.cs

```diff
@@ -295,8 +295,38 @@ internal class ExecResponseRowType
 
         [JsonProperty(PropertyName = "nullable")]
         internal bool nullable { get; set; }
+
+        [JsonProperty(PropertyName = "fields")]
+        internal List<FieldMetadata> fields { get; set; }// = new List<FieldMetadata>();
+    }
+
+    internal class FieldMetadata
+    {
+        [JsonProperty(PropertyName = "name")]
+        internal string name { get; set; }
+
+        [JsonProperty(PropertyName = "byteLength", NullValueHandling = NullValueHandling.Ignore)]
+        private Int64 byteLength { get; set; }
+
+        [JsonProperty(PropertyName = "typeName")]
+        internal string typeName { get; set; }
+
+        [JsonProperty(PropertyName = "type")]
+        internal string type { get; set; }
+
+        [JsonProperty(PropertyName = "scale", NullValueHandling = NullValueHandling.Ignore)]
+        internal Int64 scale { get; set; }
+
+        [JsonProperty(PropertyName = "precision", NullValueHandling = NullValueHandling.Ignore)]
+        internal Int64 precision { get; set; }

```

Comment:
Same here.

Reply:
maybe you are right. However ExecResponseRowType has also scale as Int64 so I would need to fix it there as well. But maybe I would leave it as is.